frontend/src/components/podcast/components/episode-table/helpers/get-columns.helper.tsx

```diff
@@ -0,0 +1,43 @@
+import { Column } from 'react-table';
+import { INCREASE_CONT_FOR_IDX } from 'common/constants/constants';
+import { Link } from 'components/common/common';
+import { AppRoute } from 'common/enums/enums';
+
+type Row = {
+  name: string,
+  episodeId: string,
+};
```

Comment:
```suggestion
type EpisodeNameRow = {
  name: string,
  episodeId: string,
};
```
move to `frontend/src/components/podcast/components/episode-table/types/`